interfaces/cython/cantera/onedim.pyx

```diff
@@ -466,6 +500,62 @@ cdef class _FlowBase(Domain1D):
     def __dealloc__(self):
         del self.flow
 
+    property settings:
+        """
+        Metadata
```

Comment:
Please add some documentation here.

Reply:
deleted the docstring here as it overloads `Domain1D.settings` (which is updated).